mesh_aws_client/tests/mesh_poll_mailbox_application_test.py

```diff
@@ -87,3 +87,6 @@ def test_mesh_poll_mailbox_happy_path(self, mock_response):
         self.assertLogs("LAMBDA0001", level="INFO")
         self.assertLogs("LAMBDA0002", level="INFO")
         self.assertLogs("LAMBDA0003", level="INFO")
+        self.assertTrue(
+            self.log_helper.was_value_logged("MESHPOLL0001", "Log_Level", "INFO")
```

Comment:
What is the difference between self.assertLogs and self.assertTrue(self.log_helper.was_value_logged ? The first is much more compact, so if it works, would it not be better to use it in all our tests?